src/js/components/Button/doc.js

```diff
@@ -134,7 +134,8 @@ export const themeDoc = {
     type: 'string | { dark: string, light: string }',
   },
   'button.disabled.opacity': {
```

Comment:
I'm thinking we shouldn't expose it at all, but still support it in the background. thoughts?

Reply:
well if we still support this i think it is better to write it down. enables people to see the deprecation wanring as well. but no strong feelings here.

Reply:
Are we sure we want to deprecate this? I do like the idea of having something more general, but I could still see a per-component style for this to customize further.

Reply:
i was thinking to deprecate this mainly to match the rest of the areas the disabled opacity will be used. I mean why would we have a button specific one but not one for textInput or textArea? Unless we do add to the other areas as well.

Reply:
I see. I think I'd tend towards each component getting their own, rather than removing it from Button.